universum/modules/structure_handler.py

```diff
@@ -140,23 +156,25 @@ def execute_one_step(self, configuration: Step,
         # step_executor is [[Step], Step], but referring to Step creates circular dependency
```

Comment:
I think step_executor should be annotated with Callable[[Step], RunningStepBase]

Reply:
The comment could be removed in this case